### DISCUSSION TITLE
Mypy exclude paths option

### DISCUSSION THREAD

**Discussion by laurentluce** [python/typing#1109] [Q&A] [answered]
Hello.  The exclude flag can be set to ignore some files from being discovered by mypy.  Is there an option to not report mypy errors for some files?  The files are discovered and type checked but the errors pointing to those files are ignored.

**Comment by hauntsaninja** [comment, marked as answer]
Yes, in your mypy config file, you can do something like:
```
[mypy-somepackage.whatever.*]
ignore_errors = True
```

If you have some code that you don't want mypy to look at at all (mypy will replace all symbols originating from this code with `Any`), you can do something like:
```
[mypy-somepackage.whatever.*]
follow_imports = skip
```

**Reply by laurentluce** [reply]
Note that the module pattern needs to be prefixed with "mypy-".

**Reply by laurentluce** [reply]
Thanks @hauntsaninja for the answer.
